### PR TITLE
Make samples.pathogen_id required.

### PR DESCRIPTION
**Pull request by jgadling**
### Summary:
- **What:** Migrate all samples w/o pathogens to have SC2 attached
- **Ticket:** [sc213489](https://app.shortcut.com/genepi/story/213489)
- **Env:** `<rdev link>`

### Checklist:
- [x] I merged latest `<base branch>`
- [ ] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/backend/database_migrations/versions/20220914_205721_make_sample_pathogen_required.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    # backpopulate pathogen fields to sars-cov-2
+    op.execute(
+        """
+        UPDATE aspen.samples
+        SET pathogen_id=(
+            SELECT id from aspen.pathogens WHERE name='SARS-CoV-2'
+        )
+        WHERE pathogen_id IS NULL
+        """
+    )
```
Comment (phoenixAja) [inline src/backend/database_migrations/versions/20220914_205721_make_sample_pathogen_required.py]: do we also need to do this for phylo_runs and phylo_trees?

Comment (jgadling) [reply src/backend/database_migrations/versions/20220914_205721_make_sample_pathogen_required.py]: Like this? 

https://github.com/chanzuckerberg/czgenepi/blob/trunk/src/backend/database_migrations/versions/20220907_225235_add_pathogens_to_all_trees.py

Comment (vincent-czi) [reply src/backend/database_migrations/versions/20220914_205721_make_sample_pathogen_required.py]: Very minor nit: feels like we should use `WHERE slug='SC2'` if that's how the other migration did it?

**Review by vincent-czi — APPROVED** [review]
Looks good!

**Pull request merged**